src/components/layout-item/layout-item.scss

```diff
@@ -0,0 +1,71 @@
+:host {
+  @apply flex flex-col flex-1;
+}
+
+.container {
```

Comment:
Should we consider a bordered option or rely on users to add that if needed?

Reply:
I think we can rely on users or where this is being slotted to handle that.